src/Components/ColorSwatch/ColorSwatch.js

```diff
@@ -1,9 +1,13 @@
 import React from 'react';
 
-const ColorSwatch = () => {
+const ColorSwatch = ({ hex, isLocked }) => {
+  const lockIcon = isLocked ? <p>Locked</p> : <p>Unlocked</p>;
```

Comment:
I like this way of showing the appropriate icons.  I've added another issue to swap the p tags out with icons once we get my code merged with the fontAwesome link.